src/components/generateKeys/GenerateKeypair.tsx

```diff
@@ -0,0 +1,182 @@
+import { useState } from 'react';
+
+interface initialErrorState {
+  status: boolean;
+  message: string;
+}
+
+function GenerateKeypair() {
+  const [storedKeypair, setStoredKeypair] = useState<boolean>(false);
+  const [isCopied, setIsCopied] = useState<boolean>(false);
+  const [error, setError] = useState<initialErrorState>({ status: false, message: '' });
+
+  const textToCopy = `Public key:
+  GDRU64ATGPDM2EVNOIVVM3P3X55LT5FLUA6XMD55IYACPDDVUWZ5BQTW
+  Secret key:
+  SBZK3IZGHIM3GP2I4K32QVSWBNVUAD2KL6VMUQOQEMXAIWNXDU7KRTAD
+  `;
```

Comment:
This doesn't have to be hardcoded. We've Stellar methods to create keys dynamically.